README.md

```diff
@@ -75,6 +102,30 @@ editorElement.disableEnvironmentBanner();
 
 For a complete version of this example, checkout this file: [public/test.html](public/test.html). It also includes another button that inserts a template in the editor to showcase the plugins.
 
+## Prosemirror
+
+The rdfa editor uses prosemirror as a base, after the `editorElement.initEditor()` function is called you will have access to the editor controller with `editorElement.controller` this is an instance of the `ProseController` class fo the [ember-rdfa-editor](https://github.com/lblod/ember-rdfa-editor)
+It provides the following methods:
+- `toggleMark(name: string, includeEmbeddedView = false)`: method which allows to enable/disable a specific mark on the current selection. Expects the name of the mark to toggle and whether the command should be applied on an embedded editor view if such a view is active.
+- `focus(includeEmbeddedView = false)`: method which allows one to focus the main editor view (or an embedded view, if such a view is active).
+- `setHtmlContent(content: string)`: sets the content of the main editor.
+- `doCommand(command: Command, includeEmbeddedView = false)`: executes a Prosemirror command (https://prosemirror.net/docs/guide/#commands) on the main editor, or when active an embedded editor instance.
+- `checkCommand(command: Command, includeEmbeddedView = false)`: checks whether a Prosemirror command may be executed.
+- `isMarkActive(mark: MarkType, includeEmbeddedView = false)`: checks whether a mark is currently active.
+- `withTransaction(callback: (tr: Transaction) => Transaction | null, includeEmbeddedView = false)`: method which allows you to apply a transaction on the main view (or currently active embedded view). When you want to apply the transaction, the callback should return a transaction object.
+- `getState(includeEmbeddedView = false)`: used to request the current state of the main editor view (or an embedded view if active).
+- `getView`: used to request the main editor view (or an embedded view if active).
+- `setEmbeddedView(view: RdfaEditorView)`: activate a specific embedded view.
+- `clearEmbeddedView`: deactive the current embedded view.
+
+Additionally, a controller provides the following attributes:
```

Comment:
```suggestion
```

These are now described above, and I would simply not mention the datastore, we don't really use it and I don't wanna get questions about it :)